src/Symfony/Component/Validator/Constraints/RangeValidator.php

```diff
@@ -76,5 +76,15 @@ public function validate($value, Constraint $constraint)
                 ->setCode(Range::TOO_LOW_ERROR)
                 ->addViolation();
```

Comment:
One more thing, should a `return` be added here? If value is below `min` you'll get two errors now, for going above `max` only one. The error for range should be enough.

Reply:
You are right, thanks !